scripts/test_collect_metrics.py

```diff
@@ -0,0 +1,71 @@
+from unittest import (
```

Comment:
I would be nice to have all tests together in the /test directory, and use the sys path stuff already in place there to make collect_metrics importable.